packages/analytics/src/api.ts

```diff
@@ -167,6 +167,40 @@ export function setCurrentScreen(
   ).catch(e => logger.error(e));
 }
 
+/**
+ * Retrieves a unique identifier for the web client.
```

Comment:
Suggest "a unique Google Analytics identifier" and link to https://developers.google.com/analytics/devguides/collection/ga4/reference/config#client_id

See `logEvent` further down in this file for how to format an external link.